tests/llm/utils/mock_utils.py

```
@@ -155,9 +155,35 @@ def load_test_cases(self) -> List[HolmesTestCase]:
-                    test_case = TypeAdapter(AskHolmesTestCase).validate_python(
-                        config_dict
-                    )
+                    try:
+                        test_case = TypeAdapter(AskHolmesTestCase).validate_python(
+                            config_dict
+                        )
+                    except Exception as e:
+                        config_file_path = test_case_folder / CONFIG_FILE_NAME
+                        # Extract invalid tags from validation error if present
+                        invalid_tags = []
+                        if hasattr(e, "errors") and callable(getattr(e, "errors")):
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)